### PR TITLE
Remove wasm

### PR DESCRIPTION
**Pull request by robbiecarlton**
The dependency on `@holo-host/wasm-key-manager` was causing down stream build issues with this module in some environments, so I removed it. That repo is still available to be imported directly.

### CODE REVIEW

File: src/deriveSeedFrom.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    //  time: 1, // the number of iterations
+    //  mem: 1024, // used memory, in KiB
+    //  hashLen: 24, // desired hash length
+    //  parallelism: 1, // desired parallelism (it won't be computed in parallel, however)
+    //  secret: new Uint8Array([...]), // optional secret data <- hha_id
+    //  ad: new Uint8Array([...]), // optional associated data
+    // }
+
+    return seed
+}
```
Comment (zo-el) [inline src/deriveSeedFrom.js]: I think I am missing something here, I am seeing everything commented here

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const {
-  KeyManager,
   deriveSeedFrom
-} = require("@holo-host/wasm-key-manager");
+} = require("./deriveSeedFrom")
+const KeyManager = require("./keyManager")
```
Comment (zo-el) [inline src/index.js]: I thought we decided not to have this implementation in the lib.

we should have have two implementation, lets just remove it and if we need it then they should use the `wasm-key-manager`

Comment (robbiecarlton) [reply src/index.js]: I was just keeping the half completed js implementations around. But I'll move them to a branch

File: package.json [hunk 3]
```diff
@@ -20,14 +20,17 @@
     "src/*"
   ],
   "dependencies": {
-    "@holo-host/wasm-key-manager": "1.0.0",
+    "@noble/ed25519": "^2.1.0",
+    "@noble/hashes": "^1.5.0",
+    "argon2-browser": "^1.18.0",
```
Comment (zo-el) [inline package.json]: we should remove the unnecessary modules too

**Review by zo-el — APPROVED** [review]
Just needs to clean up the imported libs. Approving preemptively

**Pull request merged**